receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -525,6 +525,12 @@ metrics:
   # k8s.node.condition_* metrics (k8s.node.condition_ready, k8s.node.condition_memory_pressure, etc) are controlled 
   # by node_conditions_to_report config option. By default, only k8s.node.condition_ready is enabled.
 
+  k8s.node.allocatable:
+    enabled: false
+    description: The allocatable metrics of a particular Node.
+    unit: ""
```

Comment:
Shouldnt this have a unit?

Reply:
In that case, shouldn't be different metrics? We hit the same issue in resourcequota metrics and we had to split: https://github.com/open-telemetry/semantic-conventions/issues/2076#issuecomment-2850756793

Reply:
if the code owners agree, I would wait until the semconv is clear and merged and afterwards the receiver can be adapted accordingly, anybody against this approach?